src/main/target/OMNIBUSF4/target.c

```diff
@@ -24,8 +24,23 @@
 #include "drivers/timer.h"
 #include "drivers/timer_def.h"
 
+#if defined(EXUAVF4PRO)
 const timerHardware_t timerHardware[USABLE_TIMER_CHANNEL_COUNT] = {
+    DEF_TIM(TIM10, CH1, PB8,  TIM_USE_PPM,   			   TIMER_OUTPUT_NONE,     0), // PPM
 
+    DEF_TIM(TIM3,  CH3, PB0,  TIM_USE_MOTOR,               TIMER_OUTPUT_STANDARD, 0), // S1_OUT D1_ST7
+    DEF_TIM(TIM3,  CH4, PB1,  TIM_USE_MOTOR,               TIMER_OUTPUT_STANDARD, 0), // S2_OUT D1_ST2
+    DEF_TIM(TIM2,  CH4, PA3,  TIM_USE_MOTOR,               TIMER_OUTPUT_STANDARD, 1), // S3_OUT D1_ST6
+    DEF_TIM(TIM2,  CH3, PA2,  TIM_USE_MOTOR,               TIMER_OUTPUT_STANDARD, 0), // S4_OUT D1_ST1
+	DEF_TIM(TIM5,  CH2, PA1,  TIM_USE_MOTOR,               TIMER_OUTPUT_STANDARD, 0), // S5_OUT
+	DEF_TIM(TIM1,  CH1, PA8,  TIM_USE_MOTOR,               TIMER_OUTPUT_STANDARD, 0), // S6_OUT
+
+	DEF_TIM(TIM4,  CH1, PB6,  TIM_USE_LED,                 TIMER_OUTPUT_STANDARD, 0), // LED strip
+};
+
+#else
+
+const timerHardware_t timerHardware[USABLE_TIMER_CHANNEL_COUNT] = {
```

Comment:
In the interest of being as short as possible, can you only put the `DEF_TIM()` lines into the conditional block, and leave the array definition out?